_includes/header.html

```diff
@@ -1,50 +1,5 @@
-<!-- TODO this file has become a mess, refactor it -->
-
-{% if page.bigimg or page.title %}
```

Comment:
@vsoch : Could you please add this section back in?  We're not using it yet, but I was hoping to add in a banner image (page.bigimg) in the not-too-distant future.

Reply:
If you see the comment, it was about refactoring because it's a mess. And it was a mess. Can I offer to add this back in (much more cleanly, nicely done) when you are ready to add the image?

Reply:
That would be fine - thanks!